src/libs/actions/User.ts

```diff
@@ -512,7 +498,7 @@ function subscribeToUserEvents() {
             previousUpdateID: Number(pushJSON.previousUpdateID || 0),
         };
         if (!OnyxUpdates.doesClientNeedToBeUpdated(Number(pushJSON.previousUpdateID || 0))) {
-            OnyxUpdates.apply(updates);
+            OnyxUpdates.apply(updates as any);
```

Comment:
Does this type match?
```suggestion
            OnyxUpdates.apply(updates as OnyxUpdate[]);
```

Reply:
Do you mean updates a kind of `as unknown as OnyxUpdate[]`? Just to clarify:`updates` - is an array of OnyxUpdate's?